calblueprint/impact-fund
# [navigation] revamp bottom tab bar and header

## What's new in this PR
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- Changed BottomTabNav styling to reflect the Figma
- Created custom header components and used them to wrap all screens in the app.

### Screenshots

<img src="https://github.com/calblueprint/impact-fund/assets/97428041/e35f0304-1656-4dd6-b4ea-c665ac20d982" width="240" height="540"> <img src="https://github.com/calblueprint/impact-fund/assets/97428041/12e4ee94-9c89-42f6-9abb-f51a868105ca" width="240" height="540">


## How to review
- switch around bottom tabs and make sure that the styling looks correct
- make sure header looks good and updates correctly with no weird styling issues

## Next steps
- If have time, consider bringing back the disappearing bottom tab bar.

## Relevant Links

https://reactnavigation.org/docs/bottom-tab-navigator/
https://reactnavigation.org/docs/native-stack-navigator/
https://stackoverflow.com/questions/56745881/how-to-hide-bottom-navigation-bar-on-a-specific-screen-in-react-native

CC: @ronniebeggs
